### PR TITLE
Better document --docstring-length option and add it to the readme

### PR DESCRIPTION
**Pull request by AntoineD**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -166,6 +166,8 @@ Below is the help output::
                             in a mess
       --range line line     apply docformatter to docstrings between these lines;
                             line numbers are indexed at 1
+      --docstring-length length length
```
Comment (Borda) [inline README.rst:169]: "length" seems duplicate

Comment (merwok) [reply README.rst:169]: It is a range, like `line line` just above.  That could be made clearer for sure.

Comment (AntoineD) [reply README.rst:169]: I've just blindly duplicated the doc for `--range`, what would be clearer?

Comment (merwok) [reply README.rst:169]: Having different names for the params, for example `min_length max_length`

Comment (AntoineD) [reply README.rst]: I've updated the PR with your wording.

**Pull request merged**